sharding-proxy/sharding-proxy-frontend/sharding-proxy-frontend-mysql/src/main/java/org/apache/shardingsphere/shardingproxy/frontend/mysql/command/query/binary/execute/MySQLComStmtExecuteExecutor.java

```diff
@@ -60,6 +60,8 @@
     
     private volatile boolean isQuery;
     
+    private volatile boolean hasError;
```

Comment:
I am not sure whether we should add this in MySQL frontend. I suggest PostgreSQL handle exceptions the same way an MySQL in CommandExecutorTask. What do you think?

Reply:
In PostgreSQL, if return an error response when execute sql, you should not write PostgreSQLCommandCompletePacket, so i add this flag. [diff](https://github.com/apache/incubator-shardingsphere/pull/3829/files#diff-9ed7b59514c0a038841321e90a278ddc)

Reply:
Can you go through MySQL's exception mechanism and try to abstract PostgreSQL's the same way?